stacktrace.go

```diff
@@ -237,8 +239,8 @@ func contextifyFrames(frames []Frame) []Frame {
 		switch {
 		case fileExists(frame.AbsPath):
 			path = frame.AbsPath
-		case fileExists(frame.Filename):
-			path = frame.Filename
+		case fileExists(strings.TrimPrefix(frame.AbsPath, sourceCodeLocationPrefix)):
```

Comment:
What about assigning the result to be reused in the `case` block instead of running it twice?